apps/cms/lib/repo.ex

```diff
@@ -354,4 +354,21 @@ defmodule CMS.Repo do
         end
     end
   end
+
+  @doc "Get the next n events"
+  def next_n_event_teasers(n \\ []) do
```

Comment:
Since this function seems to expect a number I don't think it makes sense to have a default value of `[]`.
That said, since it's only called once, with a value for n, maybe we don't need a default value at all!